ndarray/src/main/java/org/tensorflow/ndarray/index/Indices.java

```diff
@@ -199,21 +265,109 @@ public static Index range(long start, long end) {
    * @return index
    */
   public static Index flip() {
-    return Flip.INSTANCE;
+    return slice(null, null, -1);
   }
-  
+
   /**
-   * An index that returns elements according to an hyperslab defined by {@code start},
-   * {@code stride}, {@code count}, {@code block}. See {@link Hyperslab}.
-   * 
+   * An index that returns elements according to an hyperslab defined by {@code start}, {@code stride}, {@code count},
+   * {@code block}. See {@link Hyperslab}.
+   *
    * @param start Starting location for the hyperslab.
    * @param stride The number of elements to separate each element or block to be selected.
    * @param count The number of elements or blocks to select along the dimension.
    * @param block The size of the block selected from the dimension.
-   * 
    * @return index
    */
   public static Index hyperslab(long start, long stride, long count, long block) {
     return new Hyperslab(start, stride, count, block);
   }
+
+  /**
+   * An index that inserts a new dimension of size 1 into the resulting array.
+   *
+   * @return index
+   */
+  public static Index newAxis() {
+    return NewAxis.INSTANCE;
+  }
+
+  /**
+   * An index that expands to fill all available source dimensions. Works the same as Python's {@code ...}.
+   *
+   * @return index
+   * @see #expand()
+   */
+  public static Index ellipsis() {
+    return Ellipsis.INSTANCE;
+  }
+
+  /**
+   * An index that expands to fill all available source dimensions. Works the same as Python's {@code ...}.
+   *
+   * @return index
+   */
+  public static Index expand() {
+    return ellipsis();
```

Comment:
Like we've [discussed](https://github.com/tensorflow/java/pull/166#discussion_r553098269), we probably just want to go with one name, I'm ok with any (we don't need to be too strict to follow what the names in Python so `expand` is fine if that's what you prefer).